app/controllers/DeclarationPageController.scala

```diff
@@ -42,7 +43,8 @@ class DeclarationPageController @Inject()(declareCrystallisationService: Declare
                                           implicit val ec: ExecutionContext,
                                           declarationPageView: DeclarationPageView,
                                           authorisedAction: AuthorisedAction,
-                                          errorHandler: ErrorHandler,
+                                          implicit val validTaxYearListConnector: ValidTaxYearListConnector,
```

Comment:
we shouldn't be supplying connectors to controllers, only services. Can swap out to pass the service in instead.

Reply:
Updated